mentor_classifier_api/src/mentor_classifier_api/blueprints/trainingdata.py

```diff
@@ -11,7 +11,7 @@
 trainingdata_blueprint = Blueprint("trainingdata", __name__)
 
 
-@trainingdata_blueprint.route("/<mentor>", methods=["GET"])
+@trainingdata_blueprint.route("/<mentor>", methods=["POST"])
```

Comment:
why change this to POST? This seems wrong in terms of REST API design, might cause errors for anything that currently requests this with GET method